content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
+:guilabel:`Create and View Invoice` to create and view the invoice.
+
+.. image:: down_payment/create-invoices-popup.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (1)?
+   :alt: Create invoices pop up window in Odoo Sales.
 
-There, you can post the invoice and register the payment of your customer. But, we all know that in
-real life this flow does not happen immediately. So, for now, you can return to the sales order.
-There, you will have the possibility to see the order as a customer with the **Customer preview**
-button or to reach easily the previous draft invoice with the **Invoice** button. In any case, the
-down payment will be mentioned on both (sales order and draft invoice).
+Doing so reveals the draft invoice, which mentions the down payment.
 
-.. image:: down_payment/down_payment_3.png
+.. image:: down_payment/draft-invoice.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (2)?
+   :alt: A sample draft invoice with down payment mentioned in Odoo Sales.
 
-To complete the flow, when the customer wants to pay the rest of his sales order, you must create
-another invoice. Once again, you will have the choice to make another down payment or to deduct all
-the down payments and so, paying the rest of the invoice as a regular invoice.
+From there, the invoice can be confirmed/posted, and the payment can be registered. Return to the
+sales order, via the breadcrumbs.
 
-.. image:: down_payment/down_payment_4.png
+On the sales order, the option to view what the customer will see is available, via the
+:guilabel:`Customer Preview` smart button, and the :guilabel:`Invoice` button reveals the drafted
+invoice. In either case, the down payment is mentioned on both the sales order and draft invoice.
+
+.. image:: down_payment/down-payment-line-item.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?
+   :alt: How the down payment is mentioned on the sales order in Odoo Sales.
 
-This flow is also possible with a down payment taking into account a fixed amount.
+To confirm and register the first payment of the down payment, click the :guilabel:`Invoices` smart
+button to return to the invoice. On the :guilabel:`Customer Invoice` page, click
+:guilabel:`Confirm`, then :guilabel:`Register Payment` to open a :guilabel:`Register Payment`
+pop-up form.
+
+.. image:: down_payment/register-payment-popup.png
+   :align: center
+   :alt: How the register payment pop up looks in Odoo Sales.
+
+On this form, select the appropriate :guilabel:`Journal` and confirm that the down payment amount
+is correct in the :guilabel:`Amount` field. Then, make any other modifications to the other fields,
+if necessary, before clicking :guilabel:`Create Payment`.
+
+Odoo returns to the invoice, which now has a green `Paid` banner in the upper-right corner.
+
+.. image:: down_payment/paid-banner-invoice-1.png
```

Comment:
this image is borderline line too small...we can go up to 933px wide if we need to in order to cover larger spreads of the screen, or can crop just to show the upper portion of the invoice where the banner is 👍